DatadogCore/Sources/Core/Storage/Files/Directory.swift

```
@@ -155,7 +155,7 @@ internal struct Directory: DirectoryProtocol {
         try retry(times: 3, delay: 0.001) {
             try files().forEach { file in
                 let destinationFileURL = destinationDirectory.url.appendingPathComponent(file.name)
-                try? retry(times: 3, delay: 0.0001) {
+                try retry(times: 3, delay: 0.0001) {
```

i don't see why we should ignore the error here, looks like miss in past.